lib/active_model_serializers/railtie.rb

```diff
@@ -1,7 +1,17 @@
 require 'rails/railtie'
 
-module ActiveModel
+module ActiveModelSerializers
   class Railtie < Rails::Railtie
+    initializer 'active_model_serializers.action_controller' do
+      ActiveSupport.on_load(:action_controller) do
+        ActiveSupport.run_load_hooks(:active_model_serializers, ActiveModelSerializers)
+        include ::ActionController::Serialization
+        ActionDispatch::Reloader.to_prepare do
+          ActiveModel::Serializer.serializers_cache.clear
+        end
+      end
+    end
```

Comment:
@dgynn 

> Using an initializer, and a load hook (:action_controller), and then running another load hook (:active_model_serializers) to set the logger seems like overly complicated. Depending on load ordering, someone could have an initializer that explicitly sets ActiveModelSerializers.logger but then gets overwritten. 
> Is there any other reason to have the :active_model_serializers lazy-load hook? 

Since the logger isn't necessarily  derived from the controller, I'm just being paranoid that someone might configure the logger before the controller was loaded.  This makes sure the user has the final say.. but I could me misthinking that